src/index.ts

```diff
@@ -56,3 +57,4 @@ export {
   getComparator,
   stableSort,
 };
+export type { TableRowType, EnhancedTableDetailsRow, RendererProps, TableColumnType, SortOrder };
```

Comment:
Can we move this to line 32 just to group all the type exports.
If you can, can we sort all the exports?

Reply:
I can't group them, the linter says I have to add the word "type" to export a type `Re-exporting a type when the '--isolatedModules' flag is provided requires using 'export type'`

But I will sort them in the two groups

Reply:
i meant to say have all the export types appear together, just move it below the previous export type lines.

Reply:
```
export type { IconName } from './components/Icon/icons/';
export type { Size } from './components/Size';
export type { TableRowType, EnhancedTableDetailsRow, RendererProps, TableColumnType, SortOrder };
```